components/grade-table.js

```diff
@@ -1,26 +1,55 @@
 
 class GradeTable  {
-  constructor(tableElement) {
+  constructor(tableElement, noGradesElement) {
    this.tableElement = tableElement;
+   this.noGradesElement = noGradesElement;
   }
 
   updateGrades(grades) {
-    var tbody = this.tableElement.querySelector('tbody');
+    if (grades != 0) {
+      $('p').addClass('d-none');
+    }
+
+    else {
+      $('p').addClass('d-block');
+    }
+
     $(tbody).empty();
```

Comment:
Since you have this here you dont need it above where I commented.
Note: this must come after you declare tbody (which is currently below)